README.md

```diff
@@ -0,0 +1,41 @@
+# eslint-config-tao
```

Comment:
Can you please add a section about plugins they need to be installed by the consumer.
And maybe add `peerDependencies` for `eslint` and the `es` and  `jsdoc` plugins since they are required anyway ?
And add a line of doc about jest and svelte plugins for the svelte config

Reply:
Done.